content/server/dendrite.md

```diff
@@ -19,11 +19,11 @@ efficient and less memory-intensive but still fully functional Matrix server, th
 
 You are **not required** to run a Matrix server under a subdomain (like **matrix.example.org**), regardless of server software. You can run your server under **example.org** to ensure usernames and rooms look like `@user:example.org` and `#room:example.org` respectively.
 
-Because Matrix uses **HTTP** for transport over the SSL ports (443 and 8448), you'll have to configure NGINX for it to work. This can cause confusion, especially if you're running both a [static website](/basic/nginx/) and Matrix server under the same domain (like **example.org**).
+Because Matrix uses **HTTP** for transport over the SSL ports (443 and 8448), you'll have to configure NGINX for it to work. This can cause confusion, especially if you're running both a [static website](/server/nginx/) and Matrix server under the same domain (like **example.org**).
 
 Depending on your setup, there are 2 different configurations to achieve this:
 
-1. Your *desired* domain (**example.org**) has an [A DNS record](http://localhost:1313/basic/dns/) that already points to your desired Matrix server, so you can configure this or add to your existing NGINX static site configuration to setup Matrix.
```

Comment:
Not sure what "A DNS record" was supposed to point to. It is set to literal localhost.